CHANGELOG.md

```diff
@@ -20,6 +20,12 @@ Keep it human-readable, your future self will thank you!
 - Feature: New `Boolean1DMask` class. Enables rollout training for limited area models. [#79](https://github.com/ecmwf/anemoi-training/pulls/79)
 
 ### Fixed
+- Refactored callbacks. [#60](https://github.com/ecmwf/anemoi-training/pulls/60)
```

Comment:
This shouldn't be in the 0.2.1 release section. I believe it's related to the merge from main back into develop.